src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+const CHECK_STATUS = 'src/redux/categories/CHECK_STATUS';
+
+const initialState = [];
+/* eslint-disable*/
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.